### PR TITLE
Better icons

### PR DESCRIPTION
**Pull request by AdamSchrey**
The libertine icon for the Ubuntu Touch settings has currently a darker color than other icons and it could be a .svg file. I changed that. I also changed the normal libertine icon. You can download the images I created and do whatever you want with them and you do not need to mention my name. So, if there is something wrong with my pull request you are free to download the images and add them in another way to the project. I can also change the colors of the "libertine.png" and "libertine.svg", if you do not like it. I did not compile the project and installed it on my phone, so there is no guarantee, that it works.

### CODE REVIEW

File: snap/gui/libertine-manager-app.desktop [hunk 1]
```diff
@@ -5,7 +5,7 @@ Comment=Legacy Application Sandbox
 Exec=libertine.manager-app %U
 Terminal=false
 Type=Application
-Icon=${SNAP}/usr/share/libertine/libertine_64.png
```
Comment (Flohack74) [inline snap/gui/libertine-manager-app.desktop:8]: Is that file also needed to be deleted from the repo? (libertine_64.png) - where does it come from?

Comment (AdamSchrey) [reply snap/gui/libertine-manager-app.desktop:8]: libertine_64.png is no longer needed and is/was in libertine/data. libertine.png should replace this file. I just changed the project with libertine_64.png deleted. I also think I will not make any more changes to libertine.png.

Side Note:
The new icon was uploaded in the original resolution 500x500, what is not a common icon resolution and it is also (like the old icon) not in the folders that are recommended for icons in Debian based systems, but this shouldn't cause any problems (a lot of programs do not use the recommended folders for icons or have uncommon icon resolutions).

Comment (Flohack74) [reply snap/gui/libertine-manager-app.desktop:8]: Yeah I just wonder where is the file itself deleted, and not only references to it?

Comment (Flohack74) [reply snap/gui/libertine-manager-app.desktop:8]: Ah now I see it lol thx!

**Review by Flohack74 — APPROVED** [review]

**Pull request merged**